src/test/java/ibm/jceplus/junit/base/BaseTestDESede.java

```diff
@@ -362,23 +287,14 @@ public void testDESede_OFB_ISO10126Padding() throws Exception {
         }
     }
 
-    // --------------------------------------------------------------------------
-    //
-    //
     public void testDESede_OFB_NoPadding() throws Exception {
         encryptDecrypt("DESede/OFB/NoPadding");
     }
 
-    // --------------------------------------------------------------------------
-    //
-    //
     public void testDESede_OFB_PKCS5Padding() throws Exception {
         encryptDecrypt("DESede/OFB/PKCS5Padding");
     }
 
-    // --------------------------------------------------------------------------
-    //
-    //
     public void testDESede_PCBC_ISO10126Padding() throws Exception {
         try {
             cp = Cipher.getInstance("DESede/PCBC/ISO10126Padding", providerName);
```

Comment:
should we create a new local cipher instance rather than using the cp object from line 58?

Reply:
Same as above.